plugins/woocommerce/src/Internal/Admin/Schedulers/OrdersScheduler.php

```diff
@@ -213,7 +213,7 @@ public static function get_total_imported() {
 	 */
 	public static function possibly_schedule_import( $order_id ) {
 		if ( ! OrderUtil::is_order( $order_id, array( 'shop_order' ) ) && 'woocommerce_refund_created' !== current_filter() ) {
-			return;
+			return $order_id;
 		}
 
 		self::schedule_action( 'import', array( $order_id ) );
```

Comment:
For completeness, a `return $order_id;` should also be added at the end of the method, so that it returns the supplied order id in all cases:

```suggestion
		self::schedule_action( 'import', array( $order_id ) );
		return $order_id;
```

Reply:
This feedback seems to have been addressed. cc @Konamiman